### DISCUSSION TITLE
✅Support for ssh://

### DISCUSSION THREAD

**Discussion by Thunderhawk2001** [isontheline/pro.webssh.net#554] [Q&A]
Hi.

Have you planned to support ssh:// links?
e.g. ssh://**127.0.0.1** would open webssh and connect to **127.0.0.1**.

This could be a great addition.

**Comment by isontheline** [comment]
Hi @Thunderhawk2001 👋

There is no plan - yet - for a such feature because I need to implement first the ability to launch a connection without adding it to the database.

* Have you a use case for this feature?
* Are you wanting it on iOS or macOS?
* Are you using a lot of ssh:// links on day to day use?

Thank you a lot for your feedback 🙏

**Comment by Thunderhawk2001** [comment]
Hi, Sorry for the late reply.... work, work ^^

> Have you a use case for this feature?

Yes we have. We are using a website with a dashboard where our customers are listed with some default infos, their websites and their servers. We implemented a little button behind every server IP / Hostname to connect to this server directly via Putty (ssh://<serverip/hostname>). The only current alternative that supports this is Termius, but we do not want to use a cloud based service.


> Are you wanting it on iOS or macOS? 

We'd need it mostly for iOS/iPadOS, but a few could use it on macOS.

> Are you using a lot of ssh:// links on day to day use?

We are currently rolling out iPads for our field service staff and this could immensely help them, as they won't need to browse a huge list of entries or enter the IP manually.

**Reply by isontheline** [reply]
Thank you so much for your feedback 🙏

I will so open an issue in order to track progress.

One more question please : On Termius or PuTTY, how does user select their credentials when they click on your ssh:// links ?

**Reply by Thunderhawk2001** [reply]
Ohh... nono.  I have to thank you for considering this request :)

Putty: Some colleagues enter their username manually, as for myself: I've set up the "auto-login name" for the default profile, (as my username is always the same on every server).

But.... good point. This raises the question: What happen's, if we ever need to use different usernames. 🙈  I don't think this will happen in the near future, but who knows.... 🤔

**Reply by isontheline** [reply]
Every request need to be considered ;) and this is a good one!

"Last" question : when using ssh://links which behavior do you want?
1. Create a connection in the list (in order to be used later without links)
2. Just proceed to connection without adding the connection to the list

Thank you

**Reply by Thunderhawk2001** [reply]
Thank you so much :)

Normally I'd say: "both, with a toggle in settings" xD , but I think option 1. would be the best for our situation, as it could partialy cover the import "problem".

Have a great day/evening :)

**Reply by isontheline** [reply]
If (1.) could partial cover your import need so I will go about (1.) in the first implementation and then schedule (2.) later.

Have a nice day too 🙏. 😉
